tests/fixtures/tests.py

```diff
@@ -688,6 +688,21 @@ def test_exclude_option_errors(self):
         with self.assertRaisesMessage(management.CommandError, msg):
             management.call_command('loaddata', 'fixture1', exclude=['fixtures.FooModel'], verbosity=0)
 
+    def test_natural_keys_with_multi_table_inheritance_models(self):
```

Comment:
Changed it to `tests/serializers` and used the existing infrastructure to write a single test function and add it to all format tests. What do you think? Btw, sorry, didn't know about this infrastructure before.